### ISSUE TITLE
State attribute definition missing for : liveseg with value : -1

### ISSUE THREAD

**Issue by DcFamas** [DrozmotiX/ioBroker.wled#479]
```
wled.0 | 2022-12-28 11:02:59.160 | warn | State attribute definition missing for : liveseg with value : -1
-- | -- | -- | --
```

What can I do against this log enrty? It appears every minute. :/

**Comment by trovanaffm** [comment]
Fixed by iobroker22 on PR 471

**Comment by DutchmanNL** [comment]
released in v0.7.0
